docs/references/functions/list-builds.md

```diff
@@ -0,0 +1 @@
+Get a list of all the current user build logs. You can use the query params to filter your results. On admin mode, this endpoint will return a list of all of the project's executions. [Learn more about different API modes](/docs/admin).
```

Comment:
I'm currently 50/50 whether to keep it included, I originally added it for debugging but at the same time it could help with adding build status to the dashboard without me needing to clone the latest build information on the tag.

Reply:
I say we keep it

Reply:
Agree, but we have to document them.